cmd/query/Dockerfile

```diff
@@ -7,5 +7,6 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 
 EXPOSE 16686
 COPY query-linux /go/bin/
-ADD jaeger-ui-build /go/jaeger-ui/
-CMD ["/go/bin/query-linux", "--query.static-files=/go/jaeger-ui/"]
+
+# TODO replace this with ENTRYPOINT
```

Comment:
I did at first, but it would be a breaking change (existing docker-compose, helm/kubernetes configs won't work), so I think it's better to do for all binaries at once (#815).